outputs/core_pools.json

```diff
@@ -72,8 +69,6 @@
     "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH"
   },
   "gnosis": {
```

Comment:
Bit confused with what is going on here and what gnosis intended.  Would be good to check before we run core pools tomororw.

Reply:
ah i missed this comment. wdym "what gnosis intended"?